docs/Usage/ConfigFiles.md

```diff
@@ -19,26 +19,63 @@ You can have a user configuration file which allows you to record preferences th
 With the above user configuration file contents, Link will display notifications when processing changes to the source (notify:1), and use default to only propagating changes from the namespace to source files (watch:ns), unless a call to [Link.Create](..API/Link.Create.md) includes an explicit setting of that option .
 
 ### Directory Configuration Files
-Each linked directory may contain a `.linkconfig` file containing defaults that will apply when a link is created to that directory, or if the directory is imported. When [Link.Create](..API/Link.Create.md) or [Link.Export](..API/Link.Export.md) create a directory and create files in it, any non-default switch settings will be recorded in a configuration file within the directory. This means that you no longer need to remember the options used to re-create the original link when continuing work, or importing the link into a runtime environment.
+Each linked directory may contain a `.linkconfig` file containing defaults that will apply when a link is created to that directory, or if the directory is imported. When [Link.Create](..API/Link.Create.md) or [Link.Export](..API/Link.Export.md) create a directory and create files in it, any non-default switch settings provided to tha API function will be recorded in a configuration file within the directory. This means that you no longer need to remember the options used to re-create the original link when continuing work, or importing the link into a runtime environment.
 
 If you already have a Link folder which was created by an earlier version of Link, you can add a `.linkconfig` file using [Link.Configure](..API/Link.Configure.md). For example:
 
 ```
-      ]link.Create linkdemo c:\tmp\linkdemo
-      ]link.configure linkdemo flatten:1
+      ]Link.Create linkdemo /tmp/linkdemo
+      ]Link.Configure linkdemo flatten:1
 Was  flatten: 
 ```
+The result documents that there was no previous setting for the option. A file called `/tmp/linkdemo/.linkconfig` is created, with the following contents:
 
-This will create a file called `/tmp/linkdemo/.linkconfig` with the following contents:
+```
+{
+  Settings: {
+    flatten: 1,
+  },
+}
+```
+### Stop and Trace flags
+
+Directory configuration files also record stop and trace settings for functions and operators in the linked directory, which you can either manipulate in the editor, or using the API functions [Link.Stop](../API/Link.Stop.md) and (Link.Trace)[../API/Link.Trace.md]:
```

Comment:
second hyperlink wrong syntax: `(Link.Trace)[../API/Link.Trace.md]` should be `[Link.Trace](../API/Link.Trace.md)`